.github/workflows/release.yml

```diff
@@ -57,9 +57,41 @@ jobs:
         with:
           name: translated
           path: schemas
-  release:
+  resolve-suggestions-urls:
     needs: translate-schemas
     runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Download translated schemas
+        uses: actions/download-artifact@v1
+        with:
+          name: translated
+      - name: Install pipenv --dev
+        run: pip install pipenv==2018.11.26
+      - run: |
+          echo "::set-env name=PYTHON_VERSION::$(python -c 'import platform; print(platform.python_version())')"
+      - name: Cache virtualenv
+        id: cache-virtualenv
+        uses: actions/cache@v1
+        with:
+          path: ~/.local/share/virtualenvs/
+          key: ${{ runner.os }}-${{ env.PYTHON_VERSION }}-virtualenvs-${{ hashFiles('Pipfile.lock') }}
+      - name: Install virtual environment
+        if: steps.cache-virtualenv.outputs.cache-hit != 'true'
+        run: pipenv install --dev
+      - name: Resolve suggesions urls
+        run: make resolve-suggestions-urls
+      - name: Archive schema artifacts
+          uses: actions/upload-artifact@v1
+          with:
+            name: translated
+            path: schemas
```

Comment:
Same as comment above. If we call `make build` in the build job then that would deal with translations and resolving suggestions urls.